### PR TITLE
sso on app load

### PR DESCRIPTION
**Pull request by nameisaravind**
single sign on during app load

### CODE REVIEW

File: src/app/user/sso/sso.authguard.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+import { CanActivate, Router } from '@angular/router';
+
+@Injectable()
+export class SsoAuthGuard implements CanActivate {
+  constructor(private router: Router) {}
+
+  canActivate() {
+    if (!document.baseURI.includes('localhost')) {
+      if (!localStorage.getItem('auth-code') || !localStorage.getItem('access_token')) {
```
Comment (tatlax3636) [inline src/app/user/sso/sso.authguard.ts:10]: Do the 'auth-code' and 'access_token' need to be checked against a value? Or can they be anything?

Comment (nameisaravind) [reply src/app/user/sso/sso.authguard.ts:10]: @tatlax3636 it can be anything.

**Review by tatlax3636 — APPROVED** [review]
LGTM

**Review by vidhya9lakshmi — APPROVED** [review]
LGYM

**Pull request merged**